mmv1/products/bigquery/Dataset.yaml

```diff
@@ -26,7 +26,8 @@ docs:
     The API does accept both formats but it will always return the legacy format which results in Terraform
     showing permanent diff on each plan and apply operation.
 base_url: 'projects/{{project}}/datasets'
-self_link: 'projects/{{project}}/datasets/{{dataset_id}}'
+id_format: 'projects/{{project}}/datasets/{{dataset_id}}'
+self_link: 'projects/{{project}}/datasets/{{dataset_id}}?accessPolicyVersion=3'
```

Comment:
The [create_url](https://github.com/modular-magician/terraform-provider-google-beta/blob/f4e529fbbe377851a822114e4d6f289af060f189/google-beta/services/bigquery/resource_bigquery_dataset.go#L596) doesn't have `accessPolicyVersion=3`